tests/tests_app/core/test_lightning_flow.py

```diff
@@ -674,109 +669,3 @@ def run(self):
             assert len(self._calls["scheduling"]) == 8
 
     Flow().run()
-
-
-class WorkReload(LightningWork):
```

Comment:
Could you explain which of these tests have moved and where, and which have been removed and why?

Reply:
I removed these tests because they test these two methods `_state_dict` and `_load_state_dict` which are not used and have some bugs, that's why I put a comment above to remove them.